### ISSUE TITLE
[CI] ReloadSecureSettingsIT.testReloadAllNodesWithPasswordWithoutTLSFails timed out

### ISSUE THREAD

**Issue by droberts195** [elastic/elasticsearch#51546]
`ReloadSecureSettingsIT.testReloadAllNodesWithPasswordWithoutTLSFails` failed and caused a suite timeout in https://gradle-enterprise.elastic.co/s/ncsu2j6bxs2dc

```
com.carrotsearch.randomizedtesting.UncaughtExceptionError: Captured an uncaught exception in thread: Thread[id=2890, name=elasticsearch[transport_client_node_sm0][listener][T#6], state=RUNNABLE, group=TGRP-ReloadSecureSettingsIT] |  
-- | --
  |   |  
  | Caused by: |  
  | java.lang.AssertionError: |  
  | Expected: a string containing "Secure settings cannot be updated cluster wide when TLS for the transport layer is not enabled" |  
  | but: was "[node_sm0][127.0.0.1:40377][cluster:admin/nodes/reload_secure_settings]" |  
  | at __randomizedtesting.SeedInfo.seed([5584ECBEEA2ED9D5]:0) |  
  | at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18) |  
  | at org.junit.Assert.assertThat(Assert.java:956) |  
  | at org.junit.Assert.assertThat(Assert.java:923) |  
  | at org.elasticsearch.action.admin.ReloadSecureSettingsIT$3.onFailure(ReloadSecureSettingsIT.java:178)
```

This reproduces locally for me on the 7.x branch on Linux using:

```
./gradlew ':server:integTest' --tests "org.elasticsearch.action.admin.ReloadSecureSettingsIT.testReloadAllNodesWithPasswordWithoutTLSFails" \
  -Dtests.seed=5584ECBEEA2ED9D5 \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SA \
  -Dtests.timezone=PNT \
  -Dcompiler.java=13
```

**Comment by droberts195** [comment]
Sorry, when I first opened this issue I pasted the wrong repro command into the description.  I have edited the description now to contain the correct one.

**Comment by droberts195** [comment]
This has failed quite a few times now, so I've muted it on 7.x (only) in b8adb59e4a908d6394663054a41cb77383be21aa

**Comment by rjernst** [comment]
Fixed by #60801
